lisa/target.py

```diff
@@ -668,27 +668,32 @@ def freeze_userspace(self):
         logger = self.get_logger()
         if 'cgroups' not in self.target.modules:
             raise RuntimeError(
-                'Failed to freeze userspace. Ensure "cgroups" module is listed '
-                'among modules in target/test configuration')
+                'Failed to freeze userspace. Ensure "cgroups" devlib module is loaded.')
 
         controllers = [s.name for s in self.target.cgroups.list_subsystems()]
         if 'freezer' not in controllers:
             logger.warning('No freezer cgroup controller on target. '
```

Comment:
I know it's slightly ridiculous, but the discrepancy between the warning messages drives me nuts and I can't unsee it. Could you align them?

Reply:
me too actually, let's fully give into our OCD :)